lib/portlayer/storage/vsphere/image.go

```diff
@@ -366,7 +366,7 @@ func (v *ImageStore) scratch(ctx context.Context, storeName string) error {
 	log.Infof("Creating image %s (%s)", portlayer.Scratch.ID, imageDiskDsURI)
 
 	// Create the disk
-	vmdisk, err := v.dm.CreateAndAttach(ctx, imageDiskDsURI, "", defaultDiskSize, os.O_RDWR)
+	vmdisk, err := v.dm.CreateAndAttach(ctx, imageDiskDsURI, "", portlayer.Config.ScratchSize, os.O_RDWR)
```

Comment:
Purely for convenience, can we check that this is non zero and if it is zer, use the defaultDIskSize?  The reason I ask is the `vsphere/image_test.go` file expects scratch to be larger than 0.  And since the tests work without vic machine creating that config struct (`portlayer.Config`), they'll fail unless you want to create the config struct in the test `setup()` function and set the size.